CHANGELOG.md

```diff
@@ -6,29 +6,23 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
-### Notable changes
```

Comment:
This looks like a bad merge, these CHANGELOG entries should not be deleted

Reply:
[`10b7d34` (#2373)](https://github.com/nasa/cumulus/pull/2373/commits/10b7d34b479562248e4d6c138680fcb327939b6a)